package.json

```diff
@@ -405,6 +405,7 @@
     "@vscode/codicons": "^0.0.29",
     "@vscode/webview-ui-toolkit": "^0.9.3",
     "configparser": "^0.3.9",
+    "flatbuffers": "^2.0.7",
```

Comment:
Q. Should it be 'dependencies'..? It seems flatbuffer module is not used at runtime, it looks like its' only used to generate the 'circle_schema_generated.ts'. Additionally, if so, it looks that declaring the fixed flatbuffer version is required, when it actually includes the 'circle_schema_generated.ts' which is generated by the flatbuffer version 2.0.7.

Reply:
This is not being used at the moment but related functions will be implemented later on. We will remove this for now and add it again next time together with the associated features. Thank you@@

Reply:
When removing this one, please remove it from `infra/license/package-judgment.json` too :)

Reply:
Thanks :-D
If you will remove this part, please write down the flatbuffer version into your generated `circle_schema_generated.ts` file. And if you can, adding a used `CircleType.ts`'s hash signature or something to identify the generated file's source.

Reply:
We found that flatbuffers is now being used at circle_schema_generated.ts. Then, can we include flatbuffers in package.json at this time?

Reply:
Oops, I missed that...!
Then keep the flatbuffers package and then leave following information at the top(below copyright statement) of `circle_schema_generated.ts` :)

```
This file is generated with
  - https://github.com/Samsung/ONE/blob/8e41a5031dabc9b6d265cdf1cd8dd8bba598c614/res/CircleSchema/0.4/circle_schema.fbs
  - flatbuffer v2.0.7
```

BTW, is it right that the file is generated with https://github.com/Samsung/ONE/blob/8e41a5031dabc9b6d265cdf1cd8dd8bba598c614/res/CircleSchema/0.4/circle_schema.fbs?

Reply:
Actually we made this file with circle schema located in nnpackage directory.
https://github.com/Samsung/ONE/blob/8e41a5031dabc9b6d265cdf1cd8dd8bba598c614/nnpackage/schema/circle_schema.fbs
Would this be a problem..?

Reply:
It seems those 2 schemas are perfectly same when we check, and the versions are same.

Reply:
Fortunately, they are exactly same :)
But for next developers, please leave the link as https://github.com/Samsung/ONE/blob/8e41a5031dabc9b6d265cdf1cd8dd8bba598c614/res/CircleSchema/0.4/circle_schema.fbs !

Reply:
Okay We'll add the comment with the link you gave us. Thank you!!